docs/run/execution-modes-faq.mdx

```diff
@@ -0,0 +1,187 @@
+
+  - Sessions have greater efficiency because multiple jobs from a single algorithm run can be run sequentially without interruptions.  Additionally, you can submit jobs, check results, and submit new jobs within an active session without needing to start a new one.
+  - Batch mode has minimal delay between jobs, which can help avoid drift. Also, if you partition your workload into multiple jobs and run them in batch mode, you can get results from individual jobs.
+</details>
+
+## Sessions
+
+<details>
+  <summary>
+    What happens to my jobs if a session is closed?
+  </summary>
+
+If you are using the `Session` class in `qiskit-ibm-runtime`:
+
+  - `Session.close()` means the session no longer accepts new jobs, but existing jobs run to completion.
+  - `Session.cancel()` cancels all pending session jobs.
+
+If you are using the REST API directly:
+
+  - `PATCH /sessions/{id}` with `accepting_jobs=False` means the session no longer accepts new jobs, but existing jobs run to completion.
+  - `DELETE /sessions/{id}/close` cancels all pending session jobs.
+</details>
+
+<details>
+  <summary>
+    If I am using session mode and expect my experiment to take many hours, is there a way to ask for calibrations to happen?
+  </summary>
+
+No. On-demand calibration is not available.
+</details>
+
+<details>
+  <summary>
+    Is there an interactive timeout (ITTL) with session mode?
+  </summary>
+
+Yes. This reduces unwanted cost if a user forgets to close their session.
+</details>
+
+<details>
+  <summary>
+    How does session usage impact IBM Quantum Network members who are not billed by usage?
+  </summary>
+
+IBM Quantum Network members gain reserved capacity on IBM Quantum systems. Usage is deducted from this capacity and hubs with lower capacity have longer queueing time.
+</details>
+
+<details>
+  <summary>
+    Do I get the same parallelism in session mode that I get with batch mode?
+  </summary>
+
+Yes. If you submit multiple jobs simultaneously in a session, these jobs will run in parallel.
+</details>
+
+<details>
+  <summary>
+    Can sessions be interrupted by system jobs?
+  </summary>
+
+No.  Sessions run in dedicated mode, which means that the user has total access to the backend.  Sessions are never interrupted by system jobs, such as calibrations or software upgrades.
+</details>
+
+<details>
+  <summary>
+    Is compilation time counted as usage in sessions mode?
+  </summary>
+
+Yes.  In session mode, the entire time the session is active counts as usage.  The session becomes active when the first session job starts running.
+</details>
+
+## Batch
+
+<details>
+  <summary>
+    How many jobs run in parallel in batch mode?
+  </summary>
+
+The number of jobs running in parallel is based on the degree of parallelism configured for the backend, which is five for most backends. However, the number of concurrent jobs in an active batch could be lower because there could be other jobs already running when the batch becomes active.
```

Comment:
```suggestion
The number of jobs running in parallel is based on the degree of parallelism configured for the backend, which is five for most backends today. However, the number of concurrent jobs in an active batch could be lower because there could be other jobs already running when the batch becomes active.
```

Reply:
I can put this back if you really want it, but the problem with including words like "currently", "today", etc is that it goes stale if it changes and we forget to update it, and it implies that it will change in the future.  If we leave out "today", the information is still accurate, but leaves us free to change it in the future or not.  (and we'd want to update it if we changed from 5 tracks in the future anyway)